### PR TITLE
MINOR: [Java] Doc & javadoc improvements

### PR DESCRIPTION
**Pull request by liry**

### CODE REVIEW

File: docs/source/java/quickstartguide.rst [hunk 1]
```diff
@@ -209,7 +209,7 @@ a streaming and/or random-access fashion depending on application requirements.
 
 **Write data to an arrow file**
 
-Example: Write the dataset from the previous example to an Arrow random-access file.
+Example: Write the dataset from the previous example to an Arrow IPC File Format (random-access).
```
Comment (lidavidm) [inline docs/source/java/quickstartguide.rst:212]:
```suggestion
Example: Write the dataset from the previous example to an Arrow IPC file (random-access).
```

File: docs/source/java/quickstartguide.rst [hunk 2]
```diff
@@ -274,7 +274,7 @@ Example: Write the dataset from the previous example to an Arrow random-access f
 
 **Read data from an arrow file**
 
-Example: Read the dataset from the previous example to an Arrow random-access file.
+Example: Read the dataset from the previous example from an Arrow IPC File Format (random-access).
```
Comment (lidavidm) [inline docs/source/java/quickstartguide.rst:277]:
```suggestion
Example: Read the dataset from the previous example from an Arrow IPC file (random-access).
```

File: docs/source/java/quickstartguide.rst (hunk 1 above)
Comment (liry) [reply docs/source/java/quickstartguide.rst]: Thanks, updated

File: docs/source/java/quickstartguide.rst (hunk 2 above)
Comment (liry) [reply docs/source/java/quickstartguide.rst]: Thanks, updated

**Pull request merged**